index.js

```diff
@@ -49,8 +49,7 @@ class Toolbar {
         let $toolbar;
 
         let $defaultPlugins = plugins.default.map((Plugin) => { // eslint-disable-line no-unused-vars
-            // Render each plugin with the bound click handler
-            return <Plugin onClick={::this.handlePluginClick} />;
+            return <Plugin onClick={this.handlePluginClick.bind(this)} />;
```

Comment:
Our updated styleguide is cool with `.bind` in cases like these - so I am going to keep it. `(...args) => f(...args)` is a little verbose.